src/Webapi/Webapi__Url.re

```diff
@@ -1,3 +1,22 @@
+module SearchParams = {
```

Comment:
Might be better  to keep this as `URLSearchParams` for consistency with the underlying object name. I'm open for convincing though.

Reply:
Ah, that's true. I'll change it back to that name.